### ISSUE TITLE
Not removing unnecessary transform-style properties

### ISSUE THREAD

**Issue by himedlooff** [postcss/autoprefixer#539]
Try placing the following into the online demo:

```css
.test-1 {
    -webkit-transform-style: preserve-3d;
    -ms-transform-style: preserve-3d;
    -o-transform-style: preserve-3d;
    transform-style: preserve-3d;
}

.test-2 {
    transform-style: preserve-3d;
}
```

Here's what I'm getting back:

```css
.test-1 {
    -webkit-transform-style: preserve-3d;
    -ms-transform-style: preserve-3d;
    -o-transform-style: preserve-3d;
    transform-style: preserve-3d;
}

.test-2 {
    -webkit-transform-style: preserve-3d;
            transform-style: preserve-3d;
}
```

Shouldn't it remove the `-ms` and `-o` prefixes?

**Comment by ai** [comment]
There are no `-ms-transform-style` and `-o-transform-style` in the past. But I add it as common mistake.

**Comment by himedlooff** [comment]
Ah true! Thanks.

**Comment by himedlooff** [comment]
Bourbon adds them, I opened an issue for it.

**Comment by ai** [comment]
Done cfb7bac

**Comment by ai** [comment]
Released in 6.1
